src/store/ReposListStore/ReposListStore.ts

```diff
@@ -0,0 +1,163 @@
+import { HTTPMethod, RequestParams } from "@shared/store/ApiStore/types";
+import { GetOrganizationReposListParams } from "@store/GitHubStore/types";
+import {
+  normalizeRepoItem,
+  RepoItemApi,
+  RepoItemModel,
+} from "@store/models/gitHub";
+import {
+  CollectionModel,
+  getInitialCollectionModel,
+  linearizeCollection,
+  normalizeCollection,
+} from "@store/models/shared/collections";
+import rootStore from "@store/RootStore";
+import { Meta } from "@utils/meta";
+import { ILocalStore } from "@utils/useLocalStore";
+import {
+  action,
+  computed,
+  makeObservable,
+  observable,
+  runInAction,
+} from "mobx";
+
+import log from "../../utils/log/Logger";
+
+export interface IReposListStore {
+  _getOrganizationReposList(
+    params: GetOrganizationReposListParams
+  ): Promise<void>;
+}
+
+type PrivateFields =
+  | "_list"
+  | "_meta"
+  | "_perPage"
+  | "_searchValue"
+  | "_inputValue"
+  | "_page";
+
+export default class ReposListStore implements IReposListStore, ILocalStore {
+  destroy(): void {
+    this._list = getInitialCollectionModel();
+    this._meta = Meta.initial;
+  }
+
+  private _list: CollectionModel<number, RepoItemModel> = {
+    order: [],
+    entities: {},
+  };
+  private _meta: Meta = Meta.initial;
+  private readonly _apiStore = rootStore.apiStore;
+  private readonly _perPage: number = 12;
+  private _page: number = 1;
+  private _inputValue: string = "google";
+  private _searchValue: string = "";
+
+  constructor() {
+    makeObservable<ReposListStore, PrivateFields>(this, {
+      _list: observable.ref,
+      _meta: observable,
+      _perPage: observable,
+      _searchValue: observable,
+      _inputValue: observable,
+      _page: observable,
+      list: computed,
+      meta: computed,
+      inputValue: computed,
+      _getOrganizationReposList: action,
```

Comment:
_getOrganizationReposList не стрелочная функция, поэтому
```suggestion
      _getOrganizationReposList: action.bound,
```